src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -120,7 +120,11 @@ const ResultsPane: FC = () => {
                 />
               }
             >
-              <FluxMonacoEditor script={text} onChangeScript={setText} />
+              <FluxMonacoEditor
+                variables={[]}
```

Comment:
Rebased and found this new component.

Adding a blank variables array for now, since not entirely sure how this hooks into the existing state management (with different variable stores). Need input from @drdelambre .

Reply:
it's just in the state of this component right now, which might change as the prototype evolves, but will most likely stay there for a bit while we derisk the injection interaction and how we express the preview ramifications to the user. state is here:
https://github.com/influxdata/ui/blob/master/src/dataExplorer/components/ResultsPane.tsx#L44

Reply:
done.